app/src/main/res/layout/activity_login.xml

```diff
@@ -37,7 +36,7 @@
         android:layout_marginBottom="@dimen/margin_top_bottom"
         android:layout_marginEnd="@dimen/input_margin_start_to_end"
         android:layout_marginStart="@dimen/input_margin_start_to_end"
-        android:layout_marginTop="@dimen/input_margin_start_to_end"
+        android:layout_marginTop="@dimen/margin_top_bottom"
```

Comment:
same here, you are deleting the input_margin_start_to_end dimension in this line, why?